indexer/src/main/java/au/org/aodn/esindexer/service/StacCollectionMapperService.java

```diff
@@ -38,7 +39,6 @@
 public abstract class StacCollectionMapperService {
 
     public static final String REAL_TIME = "real-time";
```

Comment:
same here, public but not be used by external class: https://stackoverflow.com/questions/10047802/public-static-final-or-private-static-final-with-getter#comment12866979_10047926

Reply:
Same above